Tests/test_file_apng.py

```diff
@@ -689,3 +689,12 @@ def test_different_modes_in_later_frames(mode, default_image, duplicate, tmp_pat
     )
     with Image.open(test_file) as reloaded:
         assert reloaded.mode == mode
+
+
+def test_apng_repeated_seeks_give_correct_info() -> None:
+    with Image.open("Tests/images/apng/repeated_seeks_give_correct_info.png") as im:
```

Comment:
Thanks, can we concisely describe the image rather than how it's used? It may come in useful for other tests later. Maybe something like this?
```suggestion
    with Image.open("Tests/images/apng/different_durations.png") as im:
```